GradientView/GradientView.swift

```diff
@@ -10,239 +10,245 @@ import UIKit
 
 /// Simple view for drawing gradients and borders.
 @IBDesignable open class GradientView: UIView {
-
```

Comment:
Can you match the current indentation (tabs) so we can see the diff? Since it changed, it looks like you deleted every line and then added it back.